app/javascript/packages/document-capture/components/file-input.jsx

```diff
@@ -179,6 +179,19 @@ function FileInput(props, ref) {
     }
   }
 
+  /**
+   * @param {Blob|string|null|undefined} fileValue Optional array of file input accept patterns.
```

Comment:
Description's not right here, guessing some copy-paste leftovers:

```suggestion
   * @param {Blob|string|null|undefined} fileValue File for which to generate label.
```